react-common/components/share/ShareInfo.tsx

```diff
@@ -195,6 +239,14 @@ export const ShareInfo = (props: ShareInfoProps) => {
                 gifAddFrame={gifAddFrame}
                 registerSimulatorMsgHandler={registerSimulatorMsgHandler}
                 unregisterSimulatorMsgHandler={unregisterSimulatorMsgHandler} />}
+
+            {showQRCode &&
```

Comment:
nit: could we get back the onclick behavior of the qr code display here (the 'toggle back to normal share screen' / exit big qr screen)?  it feels more natural to me vs. clicking x which I would almost expect to toggle out of the whole share dialog.